### ISSUE TITLE
Setting variable to a font stack can throw false namingConvention warning

### ISSUE THREAD

**Issue by awayken** [rossPatton/stylint#49]
When using a variable to define a font stack, the way the fonts are named in the list can throw false namingConvention warnings.

fontname.styl
```stylus
// No warning
$font-stack = sans-serif

// No warning
$font-stack = helvetica, sans-serif

// Warning
$font-stack = Helvetica, sans-serif
```

fontname.json
```json
{
    "alphabetical": true,
    "borderNone": true,
    "brackets": false,
    "colons": false,
    "colors": false,
    "commaSpace": true,
    "commentSpace": false,
    "cssLiteral": false,
    "depthLimit": false,
    "duplicates": false,
    "efficient": true,
    "enforceVarStyle": false,
    "enforceBlockStyle": false,
    "extendPref": false,
    "globalDupe": false,
    "indentSpaces": 4,
    "leadingZero": true,
    "maxWarnings": 10,
    "maxWarningsKill": false,
    "mixed": false,
    "namingConvention": "lowercase-dash",
    "parenSpace": false,
    "placeholders": true,
    "semicolons": false,
    "Whitespace": true,
    "universal": true,
    "valid": false,
    "zeroUnits": true,
    "zIndexDuplicates": false,
    "zIndexNormalize": false
}
```

Results:
```bash
Warning:  preferred naming convention is lowercase-dash
File: fontname.styl
Line: 8: $font-stack = Helvetica, sans-serif
```

If I change the namingConvention to `lowercase_underscore` and change my variable name to `$font_stack`, every line throws a warning. I haven't tested the other namingConvention settings.

**Comment by awayken** [comment]
This might be related. In the following file, I get the same namingConvention issue but I'm setting a tuple instead of a font stack. Rather than make another issue, I'm including it here. Let me know if you'd like this broken out.

fontfaces.styl
```stylus
$font-faces = (Regular 400) (Medium 500)

for $face in $font-faces {
    @font-face {
        font-family: $face[ 1 ];
        font-weight: $face[ 2 ];
    }
}
```

Output:
```bash
Warning:  preferred naming convention is lowercase-dash
File: ../temp-stylus/fontfaces.styl
Line: 1: $font-faces = (Regular 400) (Medium 500)
```

**Comment by rossPatton** [comment]
not fully sure how to tackle this one without knowing the context of where a variable is used, and that would require a lot of changes to stylint. var/class/id names can be anything, and font names can be anything.

just use the `@stylint off` toggle? in my experience, seeing up your font stack is usually just done once, in one place and then used everywhere

**Comment by awayken** [comment]
The project I've been running Stylint against only has two places where I've run into this issue, so I'm willing to use the toggle technique. If I start to run into this more frequently, I might reopen this comment to further the discussion.

Thanks for looking into it!
